Independent install for each service, and documentation to accomodate.

MadcowOG opened the pull request:
Just a repeat of [this comment](https://github.com/fairyglade/ly/pull/368#issuecomment-1181358066) in the [OpenRC Support PR](https://github.com/fairyglade/ly/pull/368)

Commit:
independent install for each service, and documentation to accomodate.

oliebe reviewed (commented):
Nice. The normal `install` function still has the systemd service, is that intentional?

Commit:
Oops removed service install from install.
Also added tiny clarification of installing ly within install and installnoconf.

MadcowOG commented:
> Nice. The normal install function still has the systemd service, is that intentional?

No! I fixed it. Thanks.
